PerfettoProcessor/StackWalk.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using Utilities;
+
+namespace PerfettoProcessor
+{
+    public class StackWalkResult
+    {
+        public string[] Stack { get; set; } 
+        public string Module { get; set; }
+        public string Function { get; set; }
+    }
+
+    public class StackWalk
+    {
+        public StackWalk(IReadOnlyList<PerfettoStackProfileCallSiteEvent> stackProfileCallSiteEvents,
+                         IReadOnlyList<PerfettoStackProfileFrameEvent> stackProfileFrameEvents,
+                         IReadOnlyList<PerfettoStackProfileMappingEvent> stackProfileMappingEvents)
+        {
+            StackProfileCallSiteEvents = stackProfileCallSiteEvents;
```

Comment:
Looks like you're just indexing into these, so maybe store as arrays for better perf

Reply:
Going to punt. Doesn't seem to be a big deal but we can revisit later